common/changes/@itwin/build-tools/BenPusey-change-rush-env-variable-name_2023-08-09-18-41.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/build-tools",
+      "comment": "Changes name of environment variable that provides project root directory. Allows use of RUSHSTACK variable as well and updates README.",
+      "type": "minor"
```

everything is in lockstep in this repo , so this makes no difference

```suggestion
      "type": "none"
```